spec/ffmpeg/movie_spec.rb

```diff
@@ -248,7 +248,7 @@ module FFMPEG
         end
 
         it "should parse the bitrate" do
-          expect(@movie.bitrate).to eq(481846)
+          expect(@movie.bitrate).to eq(481836)
```

Comment:
At least add a comment to this magic number?

Reply:
It's a bitrate reported by the test video, nothing magic about it 😕

Reply:
Ah. Missed that this is in a test